maxim/logger/agno/client.py

```
@@ -0,0 +1,188 @@
+
+    def __init__(self, logger: Logger) -> None:
+        """Create a client and immediately instrument ``agno``.
+
+        Args:
+            logger: The :class:`~maxim.logger.Logger` instance to record events
+                with.
+        """
+        self._logger = logger
+        try:
+            instrument_agno(logger)
+        except Exception as e:  # pragma: no cover - runtime errors
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)